web/styles/settings.css

```diff
@@ -748,6 +748,24 @@ input[type="checkbox"] {
     margin: 20px 0 0;
 }
 
+#org-join-settings {
+    .dropdown-widget-button {
+        width: 325px;
+        color: hsl(0deg 0% 33%);
+
+        & i {
+            font-size: 10px;
+            margin-right: -3px;
```

Comment:
I think we can do this for other dropdown widgets as well. The widget for "Who can remove subscribers from this stream" setting is also different from other dropdowns on the page.

Would be good to have a discussion on CZO, but should not be a blocker for this PR.

Reply:
Yeah, this would be good to discuss in chat.zulip.org @Ujjawal3 can you start such a thread and link back to it here?

Reply:
I have started the discussion the discussion about this earlier [here](https://chat.zulip.org/#narrow/stream/101-design/topic/Large.20down.20arrow.20in.20dropdown.20list.20widget)